devtools/src/devtools.js

```diff
@@ -1,8 +1,12 @@
 import { options, Fragment, Component } from 'preact';
 
 export function initDevTools() {
-	if (typeof window != 'undefined' && window.__PREACT_DEVTOOLS__) {
-		window.__PREACT_DEVTOOLS__.attachPreact('10.4.1', options, {
+	const hook =
+		typeof window != 'undefined' &&
+		(window.__PREACT_DEVTOOLS__ || window.parent.__PREACT_DEVTOOLS__);
```

Comment:
Think it's be worth also checking window.top here? Not sure how common it is to load doubly-nested iframes.